includes/Lister/GalleryList.php

```diff
@@ -3,6 +3,8 @@
 namespace MediaWiki\Extension\DynamicPageList3\Lister;
 
 use MediaWiki\Extension\DynamicPageList3\Article;
+use PageImages\PageImages;
+use ExtensionRegistry;
```

Comment:
Move this above for alphabetical order.

Reply:
Also doesn't look like PageImages\PageImages is used so can be removed

Reply:
Done and done